Clockify.Tests/Fixtures/EnvironmentFixture.cs

```diff
@@ -26,7 +26,10 @@ private static void SetCapiKeyFromRunSettings()
             if (!string.IsNullOrEmpty(capiKey))
                 Environment.SetEnvironmentVariable(CapiKeyName, capiKey);
             else
+            {
+                Environment.SetEnvironmentVariable(CapiKeyName, "XtRJAPUG/ByUCIaC");
```

Comment:
I guess it's a mistake? 😉

Reply:
Oh yea, had some issues getting Rider to use the runsettings. Thanks for spotting it.